testing/scenario_app/ios/Scenarios/ScenariosUITests/PlatformViewGoldenTestManager.m

```diff
@@ -0,0 +1,56 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "PlatformViewGoldenTestManager.h"
+
+@interface PlatformViewGoldenTestManager ()
+
+@property(readwrite, strong, nonatomic) GoldenImage* goldenImage;
+@property(readwrite, strong, nonatomic) NSArray* launchArgs;
+
+// Maps the `identifier` to the launch args for the platform view test associated with this
+// `GoldenImage`
+@property(class, nonatomic, strong) NSDictionary* launchArgsMap;
+
+@end
+
+@implementation PlatformViewGoldenTestManager
+
+static NSDictionary* _launchArgsMap;
+
++ (void)initialize {
+  if (self == [PlatformViewGoldenTestManager class]) {
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+      [self setLaunchArgsMap:@{
+        @"platform_view" : @[ @"--platform-view" ],
```

Comment:
Nit - maybe if we defined this in the header for AppDelegate we could reuse it in both places?

Reply:
I don't think UITest can include any symbols from the app target. cc @jmagman to confirm. Not sure if there is a way to share code between app target and uitest target.